src/EnergyPlus/DXCoils.cc

```diff
@@ -12525,8 +12519,8 @@ void CalcMultiSpeedDXCoilCooling(EnergyPlusData &state,
 
     // Using/Aliasing
     using Curve::CurveValue;
-    auto &MSHPMassFlowRateHigh = state.dataHVACGlobal->MSHPMassFlowRateHigh;
-    auto &MSHPMassFlowRateLow = state.dataHVACGlobal->MSHPMassFlowRateLow;
+    Real64 MSHPMassFlowRateHigh = state.dataHVACGlobal->MSHPMassFlowRateHigh;
+    Real64 MSHPMassFlowRateLow = state.dataHVACGlobal->MSHPMassFlowRateLow;
     auto &MSHPWasteHeat = state.dataHVACGlobal->MSHPWasteHeat;
```

Comment:
Here's a bad `auto &` that I didn't fix, not sure why but it's not part of this sweep technically.